### PR TITLE
SPLAT-901: vSphere UPI zonal workflow

### PR DESCRIPTION
**Pull request by rvanderp3**

### CODE REVIEW

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/hold

placing hold pending retest after installer#6750 merges

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-upi-zones

File: ci-operator/step-registry/openshift/e2e/vsphere/upi/zones/openshift-e2e-vsphere-upi-zones-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: openshift-e2e-vsphere-upi-zones
+  steps:
+    pre:
+    - chain: upi-vsphere-pre-zones
+    test:
+    - ref: openshift-e2e-test
```
Comment (vrutkovs) [inline ci-operator/step-registry/openshift/e2e/vsphere/upi/zones/openshift-e2e-vsphere-upi-zones-workflow.yaml:7]: Is there a test which ensures machines can use all clusters in `failureDomains`?

**Comment by vrutkovs** [comment]
Other than the question above looks good, feel free to remove the hold
/approve
/lgtm
/hold

File: ci-operator/step-registry/openshift/e2e/vsphere/upi/zones/openshift-e2e-vsphere-upi-zones-workflow.yaml (hunk 1 above)
Comment (rvanderp3) [reply ci-operator/step-registry/openshift/e2e/vsphere/upi/zones/openshift-e2e-vsphere-upi-zones-workflow.yaml:7]: Not explicitly, but the terraform distributes the 4 compute nodes among the 4 failure domains.  I'll take a note to investigate this and see if we need to add a more complete test.  Thanks for reviewing the PR!

**Pull request merged**